packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -272,6 +272,25 @@ export default function MainFeedLayout({
       return null;
     }
 
+    if (user?.defaultFeedId && user.defaultFeedId !== user.id) {
+      return {
+        feedName: SharedFeedPage.MyFeed,
```

Comment:
This loads up in extension as well?

Reply:
Testing the unpacked extension in dev mode on chrome seems to do so, yes!